packages/deployer/subtasks/validate-storage.js

```diff
@@ -14,14 +13,18 @@ subtask(SUBTASK_VALIDATE_STORAGE).setAction(async (_, hre) => {
 
   const { contracts } = await getSourcesAST(hre);
 
-  let errorsFound;
-  errorsFound = findDuplicateStorageNamespaces(contracts) || errorsFound;
-  errorsFound = findRegularStorageSlots(contracts) || errorsFound;
-  errorsFound = findInvalidMutationsOnNamespaces(contracts) || errorsFound;
+  let errorsFound = [];
```

Comment:
Same as router validation

Reply:
```suggestion
  const errorsFound = [];
```

Reply:
actually, this probably could be a linter error 🤔